test-avocado/lib/host_setup.sh

```diff
@@ -28,7 +28,7 @@ HS_BASE_PCKGS="virt-deploy pystache sshpass telnet fabric python-pip avocado vir
 export HS_GRP="virtualization"
 HS_CON="-c qemu:///system"
 export HS_POOLNAME="cockpitx"
-export HS_POOLNAME_PATH=/home/$HS_POOLNAME
+export HS_POOLNAME_PATH=/opt/$HS_POOLNAME
```

Comment:
Did you intend to commit this? If so, I think it deserves to be in its own commit, since it changes all test setups.